.gitignore

```diff
@@ -50,3 +50,5 @@ out/
 **/mainGeneratedDataTemplate
 **/mainGeneratedRest
 gobblin-dist*
+gobblin.tar.gz
+masterPwd
```

Comment:
You don't need this since a test has been fixed to delete `masterPwd` after it is done.